source/features/bypass-checks.tsx

```diff
@@ -14,8 +14,10 @@ async function bypass(detailsLink: HTMLAnchorElement): Promise<void> {
 		return;
 	}
 
-	const directLink = await api.v3(`check-runs/${runId}`);
-	detailsLink.href = directLink.details_url;
+	const {details_url: detailsUrl} = await api.v3(`check-runs/${runId}`);
+	if (detailsUrl && !detailsUrl.startsWith('https://github.com/') && new URL(detailsUrl).pathname !== '/') { // Ignore links to GitHub repos or static product pages #3938
```

Comment:
I mentioned search params because they could use "http://app.code.com/?run_id=58383" and fall into this exception, but I think it's fine as a false negative, users would just be excluded from this feature rather than receive a broken link.

Reply:
I've added a check for the query string and made the conditions more readable.